Gemfile

```diff
@@ -22,6 +22,10 @@ gem 'sass'
 gem 'sinatra'
 gem 'yajl-ruby', require: 'yajl'
 
+group :development do
+  gem 'sinatra-contrib'
```

Comment:
According to https://github.com/sinatra/sinatra-contrib this has been merged into sinatra core now. My assumption is that this version still works as it's also still available as the old standalone gem, but it's not clear that any updates etc are backported to there. (Though I haven't looked closely at this, and might be misunderstanding entirely)

Reply:
Well spotted but it's just that the source code repos have been merged, not the gems.

https://rubygems.org/gems/sinatra-contrib/versions is still being released standalone and you can also see that the `sinatra` gem still [doesn't include the contrib files](https://github.com/sinatra/sinatra/blob/master/sinatra.gemspec#L10-L21).